packages/toast/test/toast.test.ts

```diff
@@ -308,4 +308,19 @@ describe('Toast', () => {
         expect(el.open).to.be.true;
         expect(closeSpy.callCount).to.equal(1);
     });
+    it('sp close button renders with static="white"', async () => {
+        const el = await fixture<Toast>(
+            html`
+                <sp-toast open>Help text.</sp-toast>
+            `
+        );
+        const renderRoot = el.shadowRoot ? el.shadowRoot : el;
+        const closeButton = renderRoot.querySelector(
+            'sp-close-button'
+        ) as CloseButton;
```

Comment:
Generally, I'd much prefer to test this sort of thing in VRTs. DOM structure and availability can more frequently change than the actual pixel delivered for it. However, being that this change was missed in the VRTs when it happened, previously, this seems like a good addition.

If anyone has thoughts on practices that would allow us to more formally respond to changes in the VRTs, it would be great to hear any suggestions that came up.